kore/src/Kore/Step/Simplification/AndTerms.hs

```diff
@@ -820,6 +821,27 @@ stringLiteralAndEqualsAssumesDifferent
   = Monad.Trans.lift $ cannotUnifyDomainValues first second
 stringLiteralAndEqualsAssumesDifferent _ _ = empty
 
+distributeAnd
```

Comment:
This should be called `orAnd` to match the names of other functions in this file.